src/test/app/Offer_test.cpp

```diff
@@ -1744,6 +1744,12 @@ class Offer_test : public beast::unit_test::suite
         auto jrr = wsc->invoke("submit", payment);
         BEAST_EXPECT(jrr[jss::status] == "success");
         BEAST_EXPECT(jrr[jss::result][jss::engine_result] == "tesSUCCESS");
+        if (wsc->version() == 2)
```

Comment:
If you go with my earlier suggestion to have the client tracka dn check the ID, all of these blocks can be safely removed. If not, with the clients defaulting to version 2, it makes more sense to do `if (BEAST_EXPECT(wsc->version() == 2))`.